CrazyCanvas/Source/ECS/Systems/Player/HealthSystem.cpp

```diff
@@ -101,6 +102,11 @@ void HealthSystem::FixedTick(LambdaEngine::Timestamp deltaTime)
 						PlayerDiedEvent diedEvent(entity);
 						EventQueue::SendEvent(diedEvent);
 					}
+					else if (healthComponent.CurrentHealth > 100)
+					{
+						LOG_INFO("PLAYER REACHED FULL HEALTH");
+						healthComponent.CurrentHealth = 100;
+					}
```

Comment:
This could be replaced with `healthComponent.CurrentHealth = std::max(healthComponent.CurrentHealth + 10, 100)` within the else-if statement at lines 90-94.

Reply:
Yes